### PR TITLE
Update README [1/2]

### PR DESCRIPTION
**Pull request by aboucaud**
Clean up of the existing README

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,6 +12,12 @@ clean: clean-ctags
 	rm -rf dist
 	find . -name "*.pyc" | xargs rm -f
 
+
+install:
```
Comment (jorisvandenbossche) [inline Makefile]: do we also want an inplace install version?

File: README.md [hunk 2]
```diff
@@ -1,577 +1,96 @@
...
- - code the event in databoard/specific/events
- - add it to fabfile.software
- - event example
-fab add_event:variable_stars
+- [`ramp-database`](ramp-database/README.md) - RAMP database module
+- [`ramp-engine`](ramp-engine/README.md) - RAMP runner service
+- [`databoard`](databoard/README.md) - RAMP frontend server
 
-## Backup
+The modules can be installed independantly but have been added to the same repository so they can be kept in sync.
```
Comment (jorisvandenbossche) [inline README.md]:
```suggestion
The modules can be installed independently but have been added to the same repository so they can be kept in sync.
```

File: databoard/README.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+cd <problem>
+jupyter nbconvert --to html <problem>_starting_kit.ipynb
+```
+
+
+## Set up the database
+
+To run the test you will need to set environment variable DATABOARD_TEST to True:
+
+    export DATABOARD_TEST=True
```
Comment (jorisvandenbossche) [inline databoard/README.md]: this was not DATABOARD_STAGE=TESTING now?

**Comment by aboucaud** [comment]
Merging the changes up to now. To be continued..

**Pull request merged**